docs/capabilities/system-open-source-health.md

```diff
@@ -30,7 +32,17 @@ The system level overview lists the third party libraries used in this system, c
 <img src="../images/system-osh-icon-legal.png" class="inline" />  **Legal licenses:** third-party dependencies may have restricted licenses, that restrict how you can use it or e.g. obliges you to publish certain source code. This should always trigger a check by a legal expert. 
 
 ### Navigating the top tiles in the overview page 
-The default view shows a vulnerability count first:
+The default views shows the Open Source Health Quality Model ratings first:
+
+<img src="../images/system-osh-quality-model-ratings.png" width="300" />
+
+You might notice that the tile does not include Stability, as this metric is not used to the purpose of rating calculations.
```

Comment:
Could you add a sentence to explain *why* it is not? Currently this reads a bit unfulfilling :)